### PR TITLE
Profile2

### PR DESCRIPTION
**Pull request by hrafov**

### CODE REVIEW

**Review by KnazevValeriy — DISMISSED** [review]

**Review by MysiukR — DISMISSED** [review]

File: src/main/java/com/epam/lab/page/ProfilePagePo.java [hunk 1]
```diff
@@ -67,10 +86,32 @@ public PageElement getWallButton() {
     public PageElement getProfileView() {
         return profileView;
     }
+    
+    public void getAndClickAllContactsButton() {
```
Comment (old-zoomer) [inline src/main/java/com/epam/lab/page/ProfilePagePo.java]: rename to clickAllContactsButtons

Comment (old-zoomer) [reply src/main/java/com/epam/lab/page/ProfilePagePo.java]: can't you just return web element from this method?

**Review by old-zoomer — CHANGES_REQUESTED** [review]

File: src/main/java/com/epam/lab/page/ProfilePagePo.java (hunk 1 above)
Comment (hrafov) [reply src/main/java/com/epam/lab/page/ProfilePagePo.java]: sorry

**Review by hrafov — COMMENTED** [review]
change WebElement

File: src/main/java/com/epam/lab/business/ProfilePageBo.java [hunk 2]
```diff
@@ -135,9 +135,10 @@ public String getFullClassname(String firstPart, String secondPart) {
     }
     
     @Step("PageElementBO Step:  get and click All Contacts Button...")
-	  public void getAndClickAllContactsButton() {
-		profilePagePo.getAndClickAllContactsButton();
-	  }
+    public void getAndClickAllContactsButton() {		
+	JavascriptExecutor js = (JavascriptExecutor) DriverManager.getDriver();
```
Comment (old-zoomer) [inline src/main/java/com/epam/lab/business/ProfilePageBo.java]: you should use page object here, not driver

Comment (hrafov) [reply src/main/java/com/epam/lab/business/ProfilePageBo.java]: Sorry - I copypasted this code, not thinking about Liskov Principle

Comment (hrafov) [reply src/main/java/com/epam/lab/business/ProfilePageBo.java]: I am working on this problem,but soo slow...

**Comment by hrafov** [comment]
I do it as list of web elements

**Review by hrafov — COMMENTED** [review]
change javascript executor to list of elements

**Review by old-zoomer — APPROVED** [review]

**Review by MysiukR — APPROVED** [review]

**Pull request merged**